exchange/src/rabbitmq/publisher/RabbitMQPublisher.cpp

```diff
@@ -56,21 +62,22 @@ RabbitMQPublisher::broadcastObUpdates(
     const std::vector<messages::ObUpdate>& updates, const std::string& ignore_uid
 )
 {
-    auto broadcastToClient = [&](const auto& client) {
-        if (client.uid == ignore_uid) {
+    auto broadcastToClient = [&](const std::pair<std::string, manager::Client>& pair) {
+        const std::string& uid = pair.first;
+        const manager::Client& client = pair.second;
+
+        if (uid == ignore_uid || !client.active) {
```

Comment:
```suggestion
        if (!client.active || uid == ignore_uid) {
```

bool checks are faster